### ISSUE TITLE
Remove users without a team from GitHub

### ISSUE THREAD

**Issue by jdno** [rust-lang/sync-team#84]
Right now, we invite new users to the `rust-lang` GitHub organization when they are added to a team. Eventually, they might stop contributing to the team and leave it again. When that happens, they are removed from the team in the [rust-lang/team](https://github.com/rust-lang/team) repo.

When a user is no longer part of any team or working group, we should also remove them from the `rust-lang` GitHub organization. This a) cleans up the members of the organization, making it easier to manage and review them. And b) frees up seats in the organization. The latter will become more relevant in the future, when admins will need to manually add seats in case we run out of our current allocation.

**Comment by me-diru** [comment]
Interesting, I was going through how membership in Rust GitHub Org works and it seems alumni are still kept in the GitHub org? 

Source: [https://forge.rust-lang.org/compiler/membership.html#alumni-status](https://forge.rust-lang.org/compiler/membership.html#alumni-status)

Maybe it has changed now, and would love to get more insights on this :D

**Comment by jdno** [comment]
Yes, right now they are still kept in the GitHub organization. But that's what we want/need to change, since it becomes unsustainable with time.

**Comment by georgegarrington** [comment]
Hey, I would like to work on this - having a look at it now

**Comment by MarcoIeni** [comment]
## Personal notes

This looks handy 😁
https://github.com/rust-lang/sync-team/blob/f235bd801a9fce0c57d5c698019cb477e03bdb90/src/github/api/write.rs#L379

**Comment by MarcoIeni** [comment]
Hey @georgegarrington thanks for your help on this 🙏
Do you want to finalize the work and raise a PR or should we assign this issue to someone else?

**Comment by MarcoIeni** [comment]
Ok, don't worry and thanks again for your help :)

If someone else wants to pick this, let us know.
Otherwise, feel free to work on it when you feel like!

**Comment by me-diru** [comment]
Hey @MarcoIeni !.
I wanna try giving this a shot :D

**Comment by MarcoIeni** [comment]
go for it 👍
